### ISSUE TITLE
error: variable has incomplete type 'struct stat64'

### ISSUE THREAD

**Issue by donomii** [saharNooby/rwkv.cpp#78]
Compile failing on M1 MacOSX, looks like there is a missing type. 

```
-- Accelerate framework found
-- CMAKE_SYSTEM_PROCESSOR: arm64
-- ARM detected
-- CMAKE_SYSTEM_PROCESSOR: arm64
-- ARM detected
-- Accelerate framework found
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/jeremyprice/git/go-rwkv.cpp/rwkv.cpp
[ 12%] Building C object ggml/src/CMakeFiles/ggml.dir/ggml.c.o
[ 25%] Linking C static library libggml.a
[ 25%] Built target ggml
[ 37%] Building CXX object CMakeFiles/rwkv.dir/rwkv.cpp.o

/git/go-rwkv.cpp/rwkv.cpp/rwkv.cpp:471:19: error: variable has incomplete type 'struct stat64'
    struct stat64 file_stat;
                  ^
/git/go-rwkv.cpp/rwkv.cpp/rwkv.cpp:471:12: note: forward declaration of 'stat64'
    struct stat64 file_stat;
           ^
```

**Comment by donomii** [comment]
Error disappears when compiling main at 9e2a0de8436a956c5ef80fbd5f4184311a7a568d

**Comment by saharNooby** [comment]
Looks like it will be fixed in #74
